app/helpers/pipelines_helper.rb

```diff
@@ -15,6 +15,18 @@ def definition_edit_text(definition, type)
     "Edit #{type}"
   end
 
+  def definition_delete_text(definition, type)
+    # rubocop:disable Rails/OutputSafety
+    if definition.shared?
+      "<p>This will <strong>NOT</strong> delete the #{type} definition '#{definition.name}'
```

Comment:
Can this be a `content_tag` or `tag` so you can remove the `rubocop:disable`?

Reply:
Yeah I was trying to do that aye, do you know if there is a way to output a content tag that has a strong tag nested inside of it?

I almost had it working with some spans but it wasn't ideal.